lib/ui/tabs/event_page.dart

```diff
@@ -69,6 +69,17 @@ class EventDetailPageState extends State<EventDetailPage> {
 
   @override
   Widget build(BuildContext context) {
+    Widget _noImage() {
+      return Container(
```

Comment:
Please remove this method `_noImage` from everywhere. 
Also, do not handle the error/placeholder image because they will be handled in another PR.

(Just a side note: declaring a function inside a function is usually not a good idea, especially in the build method which may be called repeatedly)